server/libs/userlibs.js

```diff
@@ -18,11 +19,12 @@ exports.register = function(req,res) {
 //login functionality for the app. Checks whether user exist and if exist the information provided is correct
 exports.login = function(req,res){
     let body = _.pick(req.body , ['email','password']);
```

Comment:
we don't need that you can get body.email and body.password without that

Reply:
Removed that.